Makefile

```diff
@@ -16,6 +16,7 @@ HELM_VERSION ?= 3.14.0
 GOLANGCI_VERSION ?= 1.54.2
 KIND_VERSION ?= 0.20.0
 KUBECTL_VERSION ?= 1.24.10
+GH_INSTALL_DIR := $(shell echo $$PATH | tr ':' '\n' | grep -E '^/|^/home|^/usr/local/bin' | head -n 1)
```

Comment:
nit: I think you can use `GITHUB_WORKSPACE` here, since it's [set by default](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables).